src/Symfony/Component/DependencyInjection/Secrets/JweHandler.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Secrets;
+
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
+
+final class JweHandler
+{
+    private $secretsLocation;
+
+    private $publicKey;
+
+    private $privateKeyLocation;
+
+    private $publicKeyLocation;
+
+    private $secrets;
+
+    /**
+     * @param string $fileLocation
```

Comment:
No need to add `@param` if there is no need for added documentation (because this is redundant thanks to the type-hint)

Reply:
👍